internal_handlers.go

```diff
@@ -93,3 +93,22 @@ func TranslateTenants(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, response)
 }
+
+func InternalSecretGet(c echo.Context) error {
+	secretDao, err := getSecretDao(c)
+	if err != nil {
+		return err
+	}
+
+	paramID, err := util.InterfaceToInt64(c.Param("id"))
+	if err != nil {
+		return err
```

Comment:
I think this one needs to be a `util.NewErrBadRequest`

Reply:
That is right, thanks!